File: src/test/resources/oracle/commits/rm-action-test-8187d840c38c0a1eaf8617c1290f2782084dd059/AdminAccount.java
public class AdminAccount {
    protected String username;
    private final String accessLevel = "admin";

    public AdminAccount(String username) {
        this.username = username;
    }

    public String displayName() {
        return username.trim().toUpperCase();
    }

    public boolean canManageUsers() {
        return accessLevel.equals("admin");
    }
}

File: src/test/resources/oracle/commits/rm-action-test-0a5085070f80f44c03645e1557c2bba8da3b335e/typescript/AdminAccount.ts
export class AdminAccount {
    protected username: string;
    private accessLevel = "admin";

    constructor(username: string) {
        this.username = username;
    }

    displayName(): string {
        return this.username.trim().toUpperCase();
    }

    canManageUsers(): boolean {
        return this.accessLevel === "admin";
    }
}
